docs/pages/lighting-setups.rst

```diff
@@ -0,0 +1,180 @@
+Working with Lights
+###################
+
+.. contents::
+    :class: m-block m-default
+
+This tutorial shows how to create and manipulate light setups.
+
+`Imports`_
+==========
+
+First, we import modules we will need.
+
+.. code:: py
+
+    import math
+
+    import magnum as mn
+    import numpy as np
+    from matplotlib import pyplot as plt
+
+    import habitat_sim
+    from habitat_sim.gfx import LightInfo, LightPositionModel
+    from habitat_sim.utils.common import quat_from_angle_axis, quat_to_magnum
+
+`Helper Functions and Constants`_
+=================================
+
+Next, we define some simple helper functions and constants.
+
+.. code:: py
+
+    def show_img(data):
+        plt.figure(figsize=(12, 12))
+        plt.imshow(data, interpolation='nearest')
+        plt.show()
+
+    def show_obs(sim):
+        show_img(sim.get_sensor_observations()["rgba_camera"])
+
+    def remove_all_objects(sim):
+        for id in sim.get_existing_object_ids():
+            sim.remove_object(id)
+
+    x_axis = np.array([1, 0, 0])
+    z_axis = np.array([0, 0, 1])
+    y_axis = np.array([0, 1, 0])
+
+`Simulator and Agent Configuration`_
+====================================
+
+Next, we create a simulator and place our agent in the scene.
+
+.. code:: py
+
+    # simulator configuration
+    backend_cfg = habitat_sim.SimulatorConfiguration()
+    backend_cfg.scene.id = "data/scene_datasets/habitat-test-scenes/van-gogh-room.glb"
+    backend_cfg.enable_physics = True
+
+    # agent configuration
+    sensor_cfg = habitat_sim.SensorSpec()
+    sensor_cfg.resolution = [1080, 960]
+    agent_cfg = habitat_sim.agent.AgentConfiguration()
+    agent_cfg.sensor_specifications = [sensor_cfg]
+
+    # create the simulator
+    sim_cfg = habitat_sim.Configuration(backend_cfg, [agent_cfg])
+    sim = habitat_sim.Simulator(sim_cfg)
+
+    # place our agent in the scene
+    agent_state = habitat_sim.AgentState()
+    agent_state.position = [5.0, 0.0, 1.0]
+    agent_state.rotation = quat_from_angle_axis(math.radians(70), y_axis) * quat_from_angle_axis(math.radians(-20), x_axis)
+    agent = sim.initialize_agent(0, agent_state)
+    agent_transform = agent.scene_node.transformation_matrix()
+
+`Scene Lighting`_
+=================
+
+By default, the scene will be shaded with no lights.
+
+.. code:: py
+
+    show_obs(sim)
+
+.. image:: scene_default_lighting.png
+
+To use a non-default light setup for the scene, simply use the `sceneLightSetup` field of
+`habitat_sim.SimulatorConfiguration` when creating/reconfiguring your Simulator.
+
+`Default Object Lighting`_
+==========================
+
+By default, added objects will be phong shaded with lights at the corners of the scene.
+
+.. code:: py
+
+    id_1 = sim.add_object(1)
+    sim.set_translation(agent_transform.transform_point([0.3, 0.9, -1.8]), id_1)
+
+    show_obs(sim)
+.. image:: default_object_lighting.png
```

Comment:
There needs to be a newline before, otherwise the page won't compile. Same in the four cases below.